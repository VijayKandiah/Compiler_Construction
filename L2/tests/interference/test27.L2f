(:go
  0 0
  %myV2 <- 1
  %myV <- 5
  %myV <<= %myV2
  %myV++
  rdi <- %myV
  call print 1   
  return
)
