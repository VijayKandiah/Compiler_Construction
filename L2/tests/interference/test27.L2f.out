myV2 myV r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rdi rdx rsi
myV myV2 r12 r13 r14 r15 rbp rbx
r10 myV2 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi
r11 myV2 r10 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi
r12 myV myV2 r10 r11 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi
r13 myV myV2 r10 r11 r12 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi
r14 myV myV2 r10 r11 r12 r13 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi
r15 myV myV2 r10 r11 r12 r13 r14 r8 r9 rax rbp rbx rcx rdi rdx rsi
r8 myV2 r10 r11 r12 r13 r14 r15 r9 rax rbp rbx rcx rdi rdx rsi
r9 myV2 r10 r11 r12 r13 r14 r15 r8 rax rbp rbx rcx rdi rdx rsi
rax myV2 r10 r11 r12 r13 r14 r15 r8 r9 rbp rbx rcx rdi rdx rsi
rbp myV myV2 r10 r11 r12 r13 r14 r15 r8 r9 rax rbx rcx rdi rdx rsi
rbx myV myV2 r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rcx rdi rdx rsi
rcx r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rdi rdx rsi
rdi myV2 r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdx rsi
rdx myV2 r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rsi
rsi myV2 r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx
